src/YesSql.Provider.Common/SqlBuilder.cs

```diff
@@ -198,6 +207,17 @@ public virtual string ToSqlString()
 
                 sb.Append("SELECT ");
 
+                if (_distinct != null)
```

Comment:
Looks like `_distinct` should be a bool and not a string

Reply:
Yeah, I couldn't decide on that.  I wasn't sure if some dialects used a different term or if Distinct is universal.  I'll make it a bool.